### PR TITLE
Fixes #36312 - Remove toast alert for register host validation

### PR DESCRIPTION
**Pull request by girijaasoni**
This PR is in reference to Katello PR: 
https://github.com/Katello/katello/pull/10518

This is to remove the alert toast on register host page

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
Please get used to https://cbea.ms/git-commit/#imperative for the commit message. This means `Remove alert ...` and not `Removed`.

**Review by stejskalleos — CHANGES_REQUESTED** [review]
Linter is complaining

File: webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/Actions.js [hunk 1]
```diff
@@ -9,21 +9,7 @@ import { foremanUrl } from '../../../../../foreman_tools';
 
 const Actions = ({ isLoading, isGenerating, handleSubmit, invalidFields }) => (
   <>
-    <FormGroup fieldId="actions_help" className="pf-u-pt-xl">
-      {invalidFields.length === 1 && (
```
Comment (Ron-Lavi) [inline webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/Actions.js:13]: I don't think that removing it totally is the solution, what if there's an actual error?

Comment (girijaasoni) [reply webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/Actions.js:13]: The toast is just for invalid fields which is taken care of by the form group validation.

Comment (girijaasoni) [reply webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/Actions.js:13]: > Linter is complaining

Thanks Leos, Updated it

**Review by Ron-Lavi — APPROVED** [review]
Thanks @girijaasoni ! tested together with the Katello PR

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**, works as expected

**Comment by Ron-Lavi** [comment]
@girijaasoni can we merge, or are there going to be more changes?

**Comment by stejskalleos** [comment]
@ekohl any other comments or can we merge?

**Comment by girijaasoni** [comment]
> @girijaasoni can we merge, or are there going to be more changes?

we can merge it

**Review by ekohl — APPROVED** [review]

**Pull request merged**